src/rollup/DeployHelper.sol

```diff
@@ -0,0 +1,72 @@
+
+    uint256 internal constant GASLIMIT = 100000;
+    uint256 internal constant MAXFEEPERGAS = 1000000000;
+
+    function _fundAndDeploy(
+        IInbox inbox,
+        uint256 _value,
+        address _l2Address,
+        bytes memory payload
+    ) internal {
+        uint256 submissionCost = inbox.calculateRetryableSubmissionFee(0, block.basefee);
+        inbox.createRetryableTicket{value: _value + submissionCost + GASLIMIT * MAXFEEPERGAS}({
```

Comment:
Why do you use `createRetryableTicket` with no payload + `sendL2Message`? If it's only to bypass validation could `depositEth` + `sendL2Message` be more efficient?

Reply:
oh I see, you have to fund specific account